spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/using/devtools.adoc

```diff
@@ -284,8 +284,6 @@ If you find such a problem, you need to request a fix with the original authors.
 == LiveReload
 
 The `spring-boot-devtools` module includes an embedded LiveReload server that can be used to trigger a browser refresh when a resource is changed.
-LiveReload browser extensions are freely available for Chrome, Firefox and Safari.
```

Comment:
Not sure whether it's needed to explicitly call out here that browser extensions are no longer needed and Spring Boot provides everything needed.